cypress/support/pages/homeCatalogue.pageObject.js

```diff
@@ -17,6 +17,70 @@ class HomeAndCataloguePageObject extends PageObject {
     cy.contains('.hrefch', product)
       .click();
   }
+
+  clickOnBtnAddToCart(buttonText = 'Add to cart') {
+    cy.contains('.btn', buttonText).click();
+  }
+
+  clickOnCart(cart) {
+    cy.contains('.nav-link', cart)
+      .click();
+  }
+
+  itemInCart(item) {
+    cy.get('#tbodyid')
+      .contains(item)
+      .should('be.visible');
+  }
+
+  clickOnBtnPlaceOrder(buttonText = 'Place Order') {
+    cy.contains('.btn', buttonText).click();
+  }
+
+  fillName(name) {
+    cy.get('#name')
+      .type(name);
+  }
+
+  fillCountry(country) {
+    cy.get('#country')
+      .type(country);
+  }
+
+  fillCity(city) {
+    cy.get('#city')
+      .type(city);
+  }
+
+  fillnumber(number) {
```

Comment:
Consider renaming the method `fillnumber` to `fillNumber` to maintain consistent camelCase naming convention for method names.